<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="yogita325401" name="com.tests.SearchTest" tests="5" failures="1" timestamp="14 Sep 2017 11:29:03 GMT" time="21.224" errors="0">
  <testcase name="testEmptyAddress" time="2.196" classname="com.tests.SearchTest"/>
  <testcase name="testLocationSuggestion" time="4.439" classname="com.tests.SearchTest"/>
  <testcase name="testWrongPostalCode" time="2.343" classname="com.tests.SearchTest">
    <failure type="java.lang.AssertionError" message="expected [The entered postcode does not exist or is not valid. Please check your input and try again.] but found [Die eingegebene Postleitzahl besteht nicht oder ist falsch. Bitte kontrolliere Deine Eingabe und versuche es erneut.]">
      <![CDATA[java.lang.AssertionError: expected [The entered postcode does not exist or is not valid. Please check your input and try again.] but found [Die eingegebene Postleitzahl besteht nicht oder ist falsch. Bitte kontrolliere Deine Eingabe und versuche es erneut.]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.tests.SearchTest.testWrongPostalCode(SearchTest.java:72)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testWrongPostalCode -->
  <testcase name="testResultInDropdown" time="1.089" classname="com.tests.SearchTest"/>
  <testcase name="testValidAddress" time="11.157" classname="com.tests.SearchTest"/>
</testsuite> <!-- com.tests.SearchTest -->
